### PR TITLE
added flag to toggle vye app

### PR DESCRIPTION
**Pull request by wafimohamed**
## Summary
```
As:

Frontend Dev

I need:

To add feature toggle "toggle_vye_application" to the vets-api and vets-website repos

So that:

The toggle can be placed on the new application called Verify Your Enrollment. This application is to launch in prod soon and we need to ensure it is behind a feature toggle.

Considerations:

NA

Acceptance Criteria:

Enter the acceptance criteria that will be used to test the user story.

AC 1 Update Vets-api with feature toggle "toggle_vye_application"
AC 2 Update Vets-website with feature toggle "toggle_vye_application"
AC 3 Place "toggle_vye_application" feature toggle around VYE app
Testing Considerations:
Testing will be conducted on local host and QA will be done in staging.

Other Considerations:

NA
```

- *This work is behind a feature toggle (flipper): YES/NO* YES
- *(Summarize the changes that have been made to the platform)*
 Added VYE main application Flipper/Toggle 

- *(If introducing a flipper, what is the success criteria being targeted?)*
This will enable the VYE application to go live and incorporate forthcoming changes related to DGIB. The flipper can then be removed after a target period of 8 months.
## Related issue(s)

- https://jira.devops.va.gov/browse/VYE-442

### CODE REVIEW

File: config/features.yml [hunk 1]
```diff
@@ -1518,3 +1518,6 @@ features:
     actor_type: user
     description: Use the new facilities API v2 for the Cerner CTA. Estimated time of removal June 18, 2024 after testing of deployment on staging of authenticated user.
     enable_in_development: true
+  toggle_vye_application:
+    actor_type: user
+    description: Enable VYE
```
Comment (LindseySaari) [inline config/features.yml:1523]: Could you remove the extra newline please?

Comment (wafimohamed) [reply config/features.yml]: removed

**Review by jsimonVA — APPROVED** [review]
LGTM

**Comment by ryan-mcneil** [comment]
image

**Review by ryan-mcneil — APPROVED** [review]
Strange that we can't get rid of that diff, but hopefully it doesn't cause issues.

**Review by LindseySaari — APPROVED** [review]

**Review by jsimonVA — APPROVED** [review]

**Pull request merged**